src/interfaces/IArbSys.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+/**
+ * @notice Minimal interface for interacting with Arbitrum system contracts
+ */
+interface IArbSys {
```

Comment:
We can add this as a submodule:
```
[submodule "lib/nitro-contracts"]
	path = lib/nitro-contracts
	url = https://github.com/OffchainLabs/nitro-contracts
```

Reply:
Agree but not sure if necessary since we also don't need the entire IArbSys interface, just this one func